packages/jobs/lib/routes/getHealth.ts

```diff
@@ -14,5 +14,8 @@ export const routeHandler: RouteHandler<Health> = {
     path,
     method,
     validate: (_req, _res, next) => next(), // No extra validation needed
-    handler: (_req, res) => res.status(200).json({ status: 'ok' })
+    handler: (_req, res) => {
+        // Not used in Render
```

Comment:
according to the logs this endpoint is called

Reply:
oups. I was looking at server (which is not a private service on Render and actually get an healthcheck).
It is ridiculous render doesn't provide healthcheck for private service.
Do we know how Render behaves when the server closes and stopped accepting requests?